app/configurator/configurator-state.tsx

```diff
@@ -1455,6 +1455,7 @@ const ConfiguratorStateProviderInternal = ({
                     };
                   }),
                 ],
+                activeChartKey: state.chartConfigs[0].key,
```

Comment:
It seemed to me that the activeChartKey was only used for edition mode, why do we need to save it in the database ?

Reply:
This was mostly done to keep it simpler to maintain the configurator state, as otherwise I think I needed to extract the active key when retrieving the config from database (and we have a clash between "state saved to db" type and "state before saving to db" types). But it's true that it's not necessary to store it in db, I'll remove it 👍

Reply:
It was just a question, no need to remove it if it makes things a little bit simpler, maybe a comment is sufficient ;)

Reply:
Right – I'll add a comment then 👌